### PR TITLE
C# bug

### PR DESCRIPTION
**Pull request by Justinasin**

### CODE REVIEW

**Review by Artiom171 — APPROVED** [review]

File: src/main/java/models/ArticleModel.java [hunk 1]
```diff
@@ -17,27 +17,38 @@
...
+                ResultSet rs;
+                rs = ps.executeQuery();
 
+                String title = null;
+
+                while (rs.next()) {
+                    title = rs.getString("Title");
+                }
+
+                String sql_example = "SELECT BodyHtml FROM Examples WHERE DocTopicId = ?";
```
Comment (Raimondas-Alisauskas) [inline src/main/java/models/ArticleModel.java:36]: It would be better do not leave unused code

File: web/jsp/header.jsp [hunk 2]
```diff
@@ -2,29 +2,30 @@
...
-                    <input type="text" placeholder="Search.." name="search_field">
-                    <input type="hidden"  name="pageNum" value="1">
-                </div>
-                    <button type="submit" class="searchButton"><i class="fa fa-search"></i></button>
-            </form>
+            </select>
+        </div>
+        <div class="searchField">
+            <input type="text" placeholder="Search.." name="search_field">
+            <input type="hidden" name="pageNum" value="1">
```
Comment (Raimondas-Alisauskas) [inline web/jsp/header.jsp:25]: You can delete his input.

File: src/main/java/models/ArticleModel.java (hunk 1 above)
Comment (Justinasin) [reply src/main/java/models/ArticleModel.java:36]: Can't understand which code is unused?

Comment (Justinasin) [reply src/main/java/models/ArticleModel.java:36]: Here are two quesries to get examples and title of the article. Both fields are added to object

File: web/jsp/header.jsp (hunk 2 above)
Comment (Justinasin) [reply web/jsp/header.jsp:25]: His input???

Comment (Raimondas-Alisauskas) [reply web/jsp/header.jsp:25]: input type="hidden"

File: src/main/java/models/ArticleModel.java (hunk 1 above)
Comment (Raimondas-Alisauskas) [reply src/main/java/models/ArticleModel.java:36]: Ok

**Review by Raimondas-Alisauskas — APPROVED** [review]

**Pull request merged**